src/main/webapp/app/pages/curation/review/ReviewPage.tsx

```diff
@@ -125,7 +121,7 @@ const ReviewPage: React.FunctionComponent<IReviewPageProps> = (props: IReviewPag
         isGermline,
         isAcceptAll: true,
         gene: geneData,
-        entrezGeneId: props.geneEntity.entrezGeneId,
+        entrezGeneId: geneEntity?.entrezGeneId as number,
```

Comment:
No longer using the prop entity since it's already fetched within this component.